mlperf_logging/rcp_checker/training_5.1.0/rcps_flux.json

```diff
@@ -0,0 +1,65 @@
+{
+  "flux_ref_1024": {
+    "Benchmark": "flux",
+    "Creator": "NVIDIA",
+    "When": "Reference RCPs before v5.1",
+    "Platform": "8xDGX-B200",
+    "BS": 1024,
+    "Hyperparams": {

```

Comment:
is eval frequency determined? is it frozen in the reference? would be good to log it also (so compliance enforcement, not rcp) and add it here